### PR TITLE
A test for not-implemented Jruby slice

### PR DESCRIPTION
**Pull request by Sammers21**
The PR includes:

* A java-based integration test for SubmitGem slice written in ruby
* SubmitGem slice java import improvement

Related to #13

### CODE REVIEW

**Comment by Sammers21** [comment]
@g4s8 , review the pull request please

**Comment by Sammers21** [comment]
@Guseyn , yes, the functionality is not implemented, the idea is to check if embedded JRuby code works.

**Comment by Guseyn** [comment]
@Sammers21 okay, JRuby, wow

**Comment by 0crat** [comment]
This pull request [#16](https://github.com/artipie/gem-adapter/issues/16) is assigned to @vatavuk[/z](https://www.0crat.com/u/vatavuk), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/a41df128-3742-4c4b-899c-316551c46337); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: src/test/java/com/artipie/gem/SubmitGemITCase.java [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+/**
+ * A test for gem submit operation.
+ *
+ * @since 0.2
+ */
+public class SubmitGemITCase {
+
+    @Test
+    public void submitNotImplemented() throws IOException {
+        final Vertx vertx = Vertx.vertx();
```
Comment (Vatavuk) [inline src/test/java/com/artipie/gem/SubmitGemITCase.java:47]: @Sammers21 let's extract starting of the server to a private method to make it more readable.

File: src/test/java/com/artipie/gem/package-info.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+/**
+ * Gem adapter tests.
+ *
+ * @since 0.1
```
Comment (Vatavuk) [inline src/test/java/com/artipie/gem/package-info.java:28]: @Sammers21 since `0.2`

File: src/test/java/com/artipie/gem/SubmitGemITCase.java (hunk 1 above)
Comment (Sammers21) [reply src/test/java/com/artipie/gem/SubmitGemITCase.java:47]: How would we close Vertx then?

Comment (Vatavuk) [reply src/test/java/com/artipie/gem/SubmitGemITCase.java:47]: @Sammers21 Ive missed that, ok I think we are good

**Comment by 0crat** [comment]
Job was finished in 19 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
@sereshqua[/z](https://www.0crat.com/u/sereshqua) please review this job completed by @vatavuk[/z](https://www.0crat.com/u/vatavuk), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by sereshqua** [comment]
@Vatavuk please make sure you will find at least 3 issues during next CR, thanks

@Sammers21 please make sure you start all the comments with the name of the user they are referred to, [see](https://github.com/artipie/gem-adapter/pull/16#discussion_r407861349)

**Comment by Vatavuk** [comment]
@sereshqua ok

**Comment by sereshqua** [comment]
@Sammers21 ping

**Comment by sereshqua** [comment]
@0crat quality acceptable

**Pull request merged**